index.js

```diff
@@ -28,7 +28,11 @@ module.exports = function (options, callback) {
 			skipTaskbar: true,
 			directWrite: true,
 			// Used to load the ipc module into $$electronIpc`
-			preload: path.join(__dirname, 'preload.js')
+			preload: path.join(__dirname, 'preload.js'),
+			webPreferences: {
+				webSecurity: false,
```

Comment:
I understand, the sites we are using are mostly ssl secured, but sometimes load images(tiles) from non-secure sites. I had to disable the webSecurity in my application for electron to load the tiles properly  https://fancam.com/ssl/test/2015/11/04/neal-snapshot-test/?snapshot=1  
Would you be happy to have this configurable as an option?